wave_matching/config/g_icp.yaml

```diff
@@ -0,0 +1,5 @@
+g_icp:
+  correspondenceRandomness: 10
```

Comment:
lets not mix up the cases, lets make the config reflect that class member variables 1-to-1 mapping to avoid confusion. So in this case this line should be `corr_rand: 10`.